app/screens/home/bottomTabs/Activities.tsx

```diff
@@ -384,103 +456,331 @@ const Activities = ({ route, navigation }: any): any => {
-              itemColor={headerColorBlack}
-              activatedItemColor={headerColor}
-              currentSelectedChildId={currentSelectedChildId}
-              showSelectedBracketData={showSelectedBracketData}
-              ItemTintColor={backgroundColor}
-            />
-          </View>
-
-          <DividerAct></DividerAct>
-          <ActivitiesCategories
-            borderColor={headerColor}
-            filterOnCategory={setFilteredActivityData}
-            fromPage={fromPage}
-            filterArray={filterArray}
-            onFilterArrayChange={onFilterArrayChange}
+        <KeyboardAvoidingView
+          // behavior={Platform.OS === "ios" ? "padding" : "height"}
+          behavior={Platform.OS === "ios" ? "padding" : "height"}
+          keyboardVerticalOffset={(Platform.OS === 'android') ? -200 : 0}
+          style={styles.flex1View}
+        >
+          <FocusAwareStatusBar animated={true} backgroundColor={headerColor} />
+          {/* <ScrollView nestedScrollEnabled={true}> */}
+          <TabScreenHeader
+            title={t('actScreenheaderTitle')}
+            headerColor={headerColor}
+            textColor="#000"
+            setProfileLoading={setProfileLoading}
           />
-          <DividerAct></DividerAct>
-
           <FlexCol>
-            {showNoData == true && suggestedGames?.length == 0 && otherGames?.length == 0 ?
-              <Heading4Center>{t('noDataTxt')}</Heading4Center>
-              : null}
-
-            <SectionList
-              sections={DATA}
-              // ref={flatListRef}
-              ref={(ref: any): any => (sectionListRef = ref)}
-              keyExtractor={(item: any, index: any): any => String(item?.id) + String(index)}
-              stickySectionHeadersEnabled={false}
-              // initialNumToRender={4}
-              // renderItem={({ item, title }) => <Item item={item} title={title}/>}
-              removeClippedSubviews={true} // Unmount components when outside of window 
-              initialNumToRender={4} // Reduce initial render amount
-              maxToRenderPerBatch={4} // Reduce number in each render batch
-              updateCellsBatchingPeriod={100} // Increase time between renders
-              windowSize={7} // Reduce the window size
-              // renderItem={({ item, section, index }) => <SuggestedActivities item={item} section={section.id} index={index} />}
-              renderItem={memoizedValue}
-              renderSectionHeader={({ section }): any => (
-                section.data.length > 0 ?
-                  <HeadingComponent section={section} />
-                  // <Text style={styles.header}>{section.title}</Text> 
-                  : null
-              )}
+            <View style={styles.ageBracketView}>
+              <SearchBox>
+                <OuterIconRow>
+
+                  <Pressable style={styles.pressablePadding} onPress={async (e): Promise<any> => {
+                    e.preventDefault();

```

Comment:
Is e.preventDefault() a valid method for onPress event of Pressable? 
Please consider using e.stopPropagation or remove the preventDefault method.